STEERING-COMMITTEE.md

```diff
@@ -139,24 +130,41 @@ Quorum is considered reached when at least half of the members are present.
 
 ## Getting in touch
 
-There are two ways to raise issues to the steering committee for decision:
+There are three ways to raise issues to the steering committee for decision:
 
 1. Emailing the steering committee at
    [steering@knative.team](mailto:steering@knative.team).
    This is a private discussion list to which all members of the committee have
    access.
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
+   
+3. Asking a question on the
+   [#knative-toc-sc-questions](https://cloud-native.slack.com/archives/C04LQCW0C03)
+   channel on [CNCF Slack](https://cloud-native.slack.com/).
 
 ## Composition
 
-The Steering Committee has five (5) seats. Of these five seats, four (4) are elected
-Contributing seats, open to any project contributor, and one (1) is an
-[End User Seat](#end-user-seat) appointed by the Steering Committee.
+The Steering Committee has five (7) seats. These 7 seats are composed as follows:
+
+- Four (4) are elected Contributing Seats, open to any project contributor.
+  Contributing Seats are elected every year, and serve a two-year term.  In the
+  event of a vacancy, the Steering Committee may select a replacement to serve
+  out the remainder of the term.
```

Comment:
```suggestion
- Four (4) are elected Contributing Seats, open to any project contributor.
  Contributing Seats serve a two-year term and are elected when their terms end.  In the
  event of a vacancy, the Steering Committee may select a replacement to serve
  out the remainder of the term.
```

Every year and 2 years seemed contradicting to me.

Reply:
If you mean that e.g. 2 seats are elected this year, and 2 are elected next year, we should clarify that.

Reply:
Clarified, thanks!